unit-test/js-unit-testing_get-animal/index.js

```diff
@@ -1 +1,9 @@
-export function getAnimal() {}
+export function getAnimal(animal) {
+  if (animal === "cats") {
+    return `I totally love ${animal}!`;
+  } else if (animal?.charAt(animal.length - 1) === "s") {
+    return `I like ${animal}!`;
+  } else {
+    return "I do not like animals at all!";
+  }
+}
```

Comment:
You haven't provided any information in the PR description so I don't know your intention behind this work. The syntax looks correct though. 👍

Reply:
Thanks for the Feedback. I have updated the present files and will start adding commet rpm next task onwards.